### ISSUE TITLE
Can't get process list working when run in a docker container

### ISSUE THREAD

**Issue by JamesKyburz** [google/cadvisor#1477]
I get the error

`process listing failed: failed to execute "/usr/sbin/chroot" command: exit status 1`

Here is the latest I tested in my docker compose

```
  cadvisor:
    image: google/cadvisor:v0.24.0
    cap_add:
      - all
    privileged: true
    volumes:
      - /:/rootfs:ro
      - /var/run:/var/run:rw
      - /sys:/sys:ro
      - /proc:/hostproc:rw
      - /var/lib/docker/:/var/lib/docker:ro
    restart: always
```

**Comment by cyrossignol** [comment]
I encountered this issue when running a cAdvisor container on a host machine with a non-standard `ps` executable (BusyBox) that doesn't support an option passed by cAdvisor (in my case, `stime`). 

This issue occurs because cAdvisor chroots into the `rootfs` of the host machine, that we mounted when running the container, before executing `ps`, so the output of `ps` depends on the result of the command available in the chroot environment. 

I worked around this issue by installing the `procps` package of utilities in the host OS, but this issue may be better solved by finding a more portable way for cAdvisor to pull process information.
